package/MDAnalysis/analysis/leaflet.py

```diff
@@ -71,14 +71,22 @@
 import warnings
 
 import numpy as np
-import networkx as NX
 
 from .. import core
 from . import distances
 from .. import selections
-
 from ..due import due, Doi
 
+
+# networkx is an optional import
+try:
+    import networkx as NX
+except ImportError:
```

Comment:
not sure why these aren't getting tested - it's possible networkx is getting pulled in by something else, test works fine locally

Reply:
Is this a blocker?

Reply:
Unclear, here and for the biopython removal there seems to be an issue with coverage, we have at least one runner without `networkx` (or `biopython` in the other PR) and both are uploading to codecov, but somehow this returns as uncovered.

Reply:
You can see here in the coverage report - those lines are indeed covered: https://app.codecov.io/gh/MDAnalysis/mdanalysis/pull/4331/blob/package/MDAnalysis/analysis/leaflet.py

Reply:
Thanks for checking @IAlibay